site/content/en/v1.4/tasks/operations/airgap-deployment.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Deploy Envoy Gateway in Air-Gapped Environments"
+---
+
+Deploying the Envoy Gateway in an air-gapped environment using a Helm chart requires careful configuration of the `values.yaml` file as well as adjustments when deploying a Gateway resource.
+
+You will need to specify custom image repositories for the following components in the Helm chart:
+
+- Gateway
+- Ratelimit
+
+## Gateway – `values.yaml` Configuration
+
+```yaml
+deployment:
+  envoyGateway:
+    image:
+      repository: custom-cr.internal.io/envoyproxy/gateway
+      tag: v1.4.1
+```
+
+## Ratelimit - `values.yaml` Configuration
+
+```yaml
+global:
+  images:
+    ratelimit:
+      image: custom-cr.internal.io/envoyproxy/ratelimit:master
+```
+
+## Gateway Requires a Custom EnvoyProxy Reference
+
+The Gateway must reference a custom EnvoyProxy resource that explicitly specifies the location of the distroless Envoy container image. Without this, the image will be pulled implicitly from Docker Hub.
```

Comment:
EnvoyProxy can be associated with either the `GatewayClass` to customize the envoy image for all the Gateways of a specific GatewayClass, or a `Gateway` for that Gateway only.

Can we show examples for both?